hummingbot/strategy/pure_market_making/constant_size_sizing_delegate.pyx

```diff
@@ -90,8 +90,10 @@ cdef class ConstantSizeSizingDelegate(OrderSizingDelegate):
         for active_order in active_orders:
             if active_order.is_buy:
                 has_active_bid = True
+                quote_asset_balance += float(active_order.quantity) * float(active_order.price)
```

Comment:
should we use decimal for calculations?

Reply:
Actually we need to refactor all the market classes so that it returns balance as decimal instead of float. Currently it returns float.

Reply:
is there a ticket for this?

Reply:
not sure

Reply:
can you make one?